docs/tutorials/start-dev-with-ralph.md

```diff
@@ -0,0 +1,348 @@
+- Start the LRS server: 
+
+```bash
+make run
+```
+
+## Working with Ralph's tray
+
+Ralph is distributed along with its tray (a deployable package for Kubernetes
+clusters using [Arnold](https://github.com/openfun/arnold)). If you intend to
+work on this tray, please refer to Arnold's documentation first.
+
+### Prerequisites
+
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/) (>`v.1.23.5`):
+  This CLI is used to communicate with the running Kubernetes instance you
+  will use.
+- [k3d](https://k3d.io/) (>`v.5.0.0`): This tool is used to set up
+  and run a lightweight Kubernetes cluster, in order to have a local
+  environment (it is required to complete quickstart instructions below to
+  avoid depending on an existing Kubernetes cluster).
+- [curl](https://curl.se/) is required by Arnold's CLI.
+- [gnupg](https://gnupg.org/) to encrypt Ansible vaults passwords and
+  collaborate with your team.
+
+### Create a local `k3d` cluster
+
+To create (or run) a local kubernetes cluster, we use `k3d`. The cluster's
+bootstrapping should be run _via_:
+
+```bash
+make k3d-cluster
+```
+
+> Running a k3d-cluster locally supposes that the 80 and 443 ports of your
+> machine are available, so that the ingresses created for your project
+> responds properly. If one or both ports are already used by another service
+> running on your machine, the `make k3d-cluster` command may fail.
+
+You can check that your cluster is running using the `k3d cluster` command:
+
+```bash
+k3d cluster list
+NAME     SERVERS   AGENTS   LOADBALANCER
+ralph    1/1       0/0      true
+```
+
+As you can see, we are running a single node cluster called `ralph`.
+
+### Bootstrap an Arnold project
+
+Once your Kubernetes cluster is running, you need to create a standard Arnold
+project describing applications and environments you need to deploy:
+
+```bash
+make arnold-bootstrap
+```
+
+Once bootstrapped, Arnold should have created a `group_vars` directory
+containing the following files:
+
+```bash
+tree group_vars
+group_vars
+├── common
+└── customer
+    └── ralph
+        ├── development
+        │   ├── main.yml
+        │   └── secrets
+        │       ├── databases.vault.yml
+        │       ├── elasticsearch.vault.yml
+        │       └── ralph.vault.yml
+        └── main.yml
+
+5 directories, 5 files
+```
+
+To create the LRS credentials file, you need to provide a list of accounts
+allowed to request the LRS in Ralph's vault:
+
+```bash
+# Setup your kubernetes environment
+source .k3d-cluster.env.sh
+
+# Decrypt the vault
+bin/arnold -d -c ralph -e development -- vault -a ralph decrypt
+```
+
+Edit the vault file to add a new account for the `foo` user with the `bar`
+password and a relevant scope:
+
+```yaml
+# group_vars/customer/ralph/development/secrets/ralph.vault.yml
+#
+# [...]
+#
+# LRS
+LRS_AUTH:
+  - username: "foo"
+    hash: "$2b$12$lCggI749U6TrzK7Qyr7xGe1KVSAXdPjtkMew.BD6lzIk//T5YSb72"
+    scopes:
+      - "foo_scope"
```

Comment:
```suggestion
      - "all"
```
We should provide valid scopes in the examples.

`all` would be an admin user

xAPI spec specifies `The LRS MUST assume a requested scope of "statements/write" and "statements/read/mine" if no scope is specified.`, which could be an alternative default value.